knoten/bundle.py

```diff
@@ -472,8 +472,13 @@ def compute_sigma0(V, dX, W_parameters, W_observations):
     num_observations = W_observations.shape[0]
     dof = num_observations - num_parameters
     VTPV = V.dot(W_observations).dot(V) + dX.dot(W_parameters).dot(dX)
-    sigma0 = np.sqrt(VTPV/dof)
-    return sigma0
+    
+    if (dof > 0):
+        sigma0 = VTPV/dof
```

Comment:
Are the results still valid if dof <= 0?  Might make sense to just throw an error here

Reply:
Just matching ISIS [implementation](https://github.com/DOI-USGS/ISIS3/blob/12e0ec6db5ea792303c13327096e73be009f967c/isis/src/control/objs/BundleResults/BundleResults.cpp#L861C1-L877C4). I did ran across a scenario that the dof was negative.